pkgs/applications/networking/cluster/cert-manager-ctl/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib, buildGoModule, fetchFromGitHub, installShellFiles }:
+
+buildGoModule rec {
+  pname = "cert-manager-ctl";
+  version = "1.6.1";
+
+  src = fetchFromGitHub {
+    owner = "jetstack";
+    repo = "cert-manager";
+    rev = "v${version}";
+    sha256 = "0zi3wmjdp0zklyv1q3zcfpqhn0jb1xj03vl0mcpx6mjb7mqzvrmd";
+  };
+
+  vendorSha256 = "sha256-xLjYfnvrfAhtXsQT6+IabAcI+cZozABB3refAbX8jrE=";
+  runVend = true;
+
+  postPatch = ''
+    substituteInPlace cmd/ctl/pkg/build/commands/commands.go \
+      --replace 'var registerCompletion = "false"' 'var registerCompletion = "true"'
```

Comment:
This looks like it can be set via `ldflags`.

https://github.com/jetstack/cert-manager/blob/ee8ec69fadff165afa96c2dd22264c16fdb7d065/make/cmctl.mk#L1

Reply:
Thanks. Done.